src/System.IdentityModel.Tokens.Jwt/ClaimsKnowledge.tti

```diff
@@ -0,0 +1,141 @@
+﻿<#+
+class ClaimsKnowledge
```

Comment:
This class is not product code, but control code to generate the product code.
This data structure describes the claims information.

Reply:
I am not understanding why we need a template model.
I have used .tt's when using typescript that will generate different jscript mapped to a release.

How does the tti help us?

Reply:
@jmprieur i will educate myself about this technique.

Reply:
I can spend 10 minutes with you @brentschmaltz if you wish. Also you might want to install this Visual Studio extension: https://marketplace.visualstudio.com/items?itemName=bricelam.T4Language

Reply:
@jmprieur Would this require users to install this extension to build?

Reply:
no. not at all. It's an editor extension so that the syntax coloring help understanding the T4 code